h2o-py/h2o/model/model_base.py

```diff
@@ -1713,7 +1713,15 @@ def score_history(self):
         """DEPRECATED. Use :meth:`scoring_history` instead."""
         return self.scoring_history()
 
+    def rule_importance(self):
```

Comment:
do we need to put this here? I think for python it should be possible to add only the RuleFitEstimator class via custom snippet

Reply:
e0efd1a